src/ui/parts/SpriteInfoPart.as

```diff
@@ -75,13 +75,13 @@ public class SpriteInfoPart extends UIPart implements DragClient {
 	}
 
 	public static function strings():Array {
-		return ['direction:', 'rotation style:', 'can drag in player:', 'show:'];
+		return ['direction:', 'rotation style:', 'can drag in fullscreen:', 'show:'];
```

Comment:
this shouldn't actually be changed. 'player' is anything but editor mode and doesn't have to be full screen..